Svc/Version/test/int/test_cmd_version.py

```diff
@@ -0,0 +1,30 @@
+""" test_cmd_version.py:
+
+Test the command version with basic integration tests.
+"""
+
+
+def test_send_command(fprime_test_api):
+    """Test that commands may be sent
+
+    Tests command send, dispatch, and receipt using send_and_assert command with a pair of version commands.
+    """
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.Version') + '.' + 'VERSION',["PROJECT"], max_delay=1)
```

Comment:
This looks good. I would like to add a check that the event reports the expected fsw version and that the telemetry channel matches too.

Same for the other commands like `FRAMEWORK`, `LIBRARY`, `CUSTOM`, `ALL`